tests/Commands/ExpireUnusedTokensCommandTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * This file is part of the Laravel-Doctrine-Sanctum project.
+ * (c) Ricardo Mosselman <mosselmanricardo@gmail.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Commands;
+
+use Bolivir\LaravelDoctrineSanctum\Commands\DeleteUnusedTokensCommand;
+use Bolivir\LaravelDoctrineSanctum\Repository\AccessTokenRepository;
+use Bolivir\LaravelDoctrineSanctum\Repository\IAccessTokenRepository;
+use Doctrine\ORM\AbstractQuery;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\QueryBuilder;
+use Illuminate\Console\OutputStyle;
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Tests\Bolivir\LaravelDoctrineSanctum\Fixtures\TestToken;
+use Tests\Bolivir\LaravelDoctrineSanctum\TestCase;
+
+class ExpireUnusedTokensCommandTest extends TestCase
```

Comment:
is het niet `DeleteUnusedTokensCommandTest`?

Reply:
Correct